test/test_genome.py

```diff
@@ -212,7 +344,7 @@ def objective(ind):
 
 
 def test_is_gene_in_input_region(rng_seed):
-    genome = gp.Genome(2, 1, 2, 1, None, [gp.Add])
+    genome = gp.Genome(2, 1, 2, 1, ID_UNUSED_GENE, [gp.Add])
```

Comment:
```suggestion
    genome = gp.Genome(2, 1, 2, 1, None, [gp.Add])
```

a bit too much search & replace? ;) this is the value for `levels_back`, also applies below!

Reply:
Yes, fixed.